src/context/reducers/dicomImagesReducer.tsx

```diff
@@ -13,6 +13,7 @@ export interface DcmImage {
   StudyDescription: string,
   SeriesInstanceUID: string,
   SeriesDescription: string,
+  StudyDate: string,
```

Comment:
Related comment: https://github.com/darwinai/covidnet_ui/pull/49/files#r616263856
Please, ensure to merge the fix into this branch